mix.exs

```diff
@@ -67,13 +66,8 @@ defmodule LiveSvelte.MixProject do
 
   defp aliases do
     [
-      "assets.build": ["esbuild module", "esbuild cdn", "esbuild cdn_min", "esbuild main"],
-      "assets.watch": [
-        "esbuild module --watch",
-        "esbuild cdn --watch",
-        "esbuild cdn_min --watch",
-        "esbuild main --watch"
-      ]
+      "assets.build": ["cmd --cd assets node build.js"],
```

Comment:
While working off this branch I noticed when installing in a brand new Phoenix project the build mustn't run? When trying to import `live_svelte`, it can't find the module because  the build artifacts don't exist `./priv/static/live_svelte.esm.js`
<img width="1311" alt="Screenshot 2024-11-25 at 8 55 35 PM" src="https://github.com/user-attachments/assets/ae0dab67-1768-42b6-8ce5-da22c41caf20">

Reply:
When using the version number (currently `0.15.0-rc4`) it should work properly since I build the files and publish them as a package to hex. So I think this only applies when you're using the github branch. Not sure if this is really an issue. Also I think this might have been an issue in the previous version too, not specific to the Svelte 5 release *I think*.

Reply:
Ahhh, makes sense. Thanks for the clarification.

Reply:
I remember now why I was the branch, I used my own fork to rebase master branch because this PRs branch doesn't include this commit 97ed1361b76aa5c243b0ec9c275acfd8689bd9b5 from master. Which seems to resolve another issue where `mix live_svelte.setup` seems to have issues finishing correctly. It looks as if I am missing `package.json`. 
Should you rebase and release a new RC?

<img width="1161" alt="Screenshot 2024-11-26 at 8 55 02 PM" src="https://github.com/user-attachments/assets/d4cfa3ff-32e4-4b71-b636-ea053daf9d70">
<img width="666" alt="Screenshot 2024-11-26 at 9 01 26 PM" src="https://github.com/user-attachments/assets/5ccb3610-e59c-4446-b71f-92ff0eee1d7b">

Reply:
Just noticed this [comment](https://github.com/woutdp/live_svelte/issues/140#issuecomment-2486323761) in this open issue. So I think to use the RC on a new project we will need the rebase with the fix.

Reply:
Makes sense! 
Just released `0.15.0-rc.5`